src/components/MultiStepForm/Summary/Summary.tsx

```diff
@@ -0,0 +1,63 @@
+import { IAddOn } from '../IFormInput';
+import { SummaryDiv, TotalDiv, SummaryRow, ChangePlanButton, HorizontalRule, TotalPrice } from './SummaryStyles';
+import { OfferHeader, OfferParagraph } from '../PlanSelection/PlanSelectionStyles';
+
+import { useNavigate } from 'react-router-dom';
+
+interface SummaryProps {
+  watch: (field: string) => any;
+  isYearlyPlan: boolean;
+}
+
+const Summary: React.FC<SummaryProps> = ({ isYearlyPlan, watch }) => {
+  const watchShowAddOns = watch('addOns');
+  const watchShowPlan = watch('selectedPlan');
+  const planType = isYearlyPlan;
```

Comment:
`isYearlyPlan` is a boolean. `planType` doesn't hear like a boolean name.
But anyways, you're gonna be changing this somewhere above